pkgs/tools/misc/kronometer/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  kdeDerivation, kdeWrapper, fetchurl, lib,
+  ecm, kdoctools,
+  kconfig, kinit
+}:
+
+let
+  unwrapped = let
+    pname = "kronometer";
+    version = "2.1.0";
+  in kdeDerivation rec {
+    name = "${pname}-${version}";
+
+    src = fetchurl {
+      url = "mirror://kde/stable/${pname}/${version}/src/${name}.tar.xz";
+      sha256 = "1nh7y4c13rscy55f5n8s2v8jij27b55rwkxh9g8r0p7mdwmw8vri";
+    };
+
+    meta = {
+      license = with lib.licenses; [ gpl2 lgpl21 fdl12 ];
```

Comment:
It's not a well documented feature, but yes. AFAIK, Hydra is the only thing that cares, and Hydra will [accept a list of licenses](https://github.com/NixOS/hydra/blob/ac23bd1539d222ceda5ec85c054454df2c25a799/src/hydra-eval-jobs/hydra-eval-jobs.cc#L126). It's not commonly used except in the other Qt/KDE packages because I think most other projects release things under one license.